data/json/items/generic/toys_and_sports.json

```diff
@@ -244,5 +244,65 @@
     "volume": "250 ml",
     "bashing": 7,
     "to_hit": 2
+  },
+  {
+    "type": "GENERIC",
+    "id": "mannequin",
+    "symbol": "@",
+    "color": "dark_gray",
+    "name": { "str": "mannequin" },
+    "category": "other",
+    "description": "A figure of an adult human, as seen in the display case at the mall. It might be able to fool some zombies if it could attract their attention.",
+    "price": 15000,
+    "price_postapoc": 500,
+    "material": [ "wood" ],
+    "weight": "81500 g",
+    "volume": "62500 ml",
+    "bashing": 18,
+    "to_hit": -2
+  },
+  {
+    "type": "TOOL",
+    "id": "mannequin_decoy",
+    "symbol": "@",
+    "color": "light_gray",
+    "name": { "str": "mannequin decoy" },
+    "category": "other",
+    "description": "A figure of an adult human, as seen in the display case at the mall. This one is cradling a talking doll. It can make sound when powered by batteries and can be used to distract some zombies.",
```

Comment:
```suggestion
    "description": "A figure of an adult human, as seen in the display case at the mall.  This one is cradling a talking doll.  It can make sound when powered by batteries and can be used to distract some zombies.",
```